### ISSUE TITLE
Connection to Postgresql fails to translate host name

### ISSUE THREAD

**Issue by photodude** [travis-ci/travis-ci#6667]
connection to Postgresql fails to translate host name on HHVM Trusty builds 

```php
PDOException: [112]: could not translate host name "localhost port=5432 dbname=joomla_ut" to address: Name or service not known
```
Looks like travis ci can't resolve `localhost` for Postgresql anymore. Possibly indicates an issue with the  /etc/hosts file.

example builds with the issue
- https://travis-ci.org/joomla/joomla-cms/jobs/160605806
- https://travis-ci.org/photodude/joomla-cms/jobs/164685871

**Comment by photodude** [comment]
I was thinking this was an environment issue, but I was unable to replicate with PHP 5.6 on trusty.

**Comment by photodude** [comment]
Unless this is somehow connected to the changes related to #6616...

**Comment by photodude** [comment]
Probably not related to #6616 I found an older build with this same issue

**Comment by photodude** [comment]
This was definitely a HHVM issue, as they had an issue in Parsing the DSN information to get the hostname. Should be fixed in HHVM 3.16 possibly in 3.15.2
